src/Adapter/Attribute/AdminAttributeGeneratorControllerWrapper.php

```diff
@@ -76,28 +84,28 @@ public function processGenerate($product, $options)
      */
     public function ajaxProcessDeleteProductAttribute($idAttribute, $idProduct)
     {
-        if (!\CombinationCore::isFeatureActive()) {
+        if (!Combination::isFeatureActive()) {
             return false;
         }
 
-        if ($idProduct && \ValidateCore::isUnsignedId($idProduct) && \ValidateCore::isLoadedObject($product = new \ProductCore($idProduct))) {
-            if (($depends_on_stock = \StockAvailableCore::dependsOnStock($idProduct)) && \StockAvailableCore::getQuantityAvailableByProduct($idProduct, $idAttribute)) {
+        if ($idProduct && Validate::isUnsignedId($idProduct) && Validate::isLoadedObject($product = new Product($idProduct))) {
+            if (($depends_on_stock = \StockAvailableCore::dependsOnStock($idProduct)) && StockAvailable::getQuantityAvailableByProduct($idProduct, $idAttribute)) {
```

Comment:
One of them was forgotten here:
`\StockAvailableCore::dependsOnStock($idProduct)`